source/trackers.json

```diff
@@ -937,7 +937,10 @@
         "snapcraftcontent.com": "ubuntu",
         "ubuntu.com": "ubuntu",
         "ubuntucompanyservices.co.za": "ubuntu",
-        "unityads.unity3d.com": "unity_ads",
```

Comment:
this is correct, as long as the `unity_ads` tracker exists
`"unityads.unity3d.com": "unity_ads",`